### PR TITLE
CmsKit - Add EntityType configuration to Comments

### LINKED ISSUE #7886: CMS Kit - Add CMSKitCommentOptions and Make EntityTypes Configurable
Ensure that all websites using the comment feature get updated.

### PR DESCRIPTION
**Pull request by enisn**
## Summary
This PR includes EntityType configuration for Comments but authorization check, becuase it has its own authorization logic on management side and doesn't require permission on public side.

---

- Closes #7886

### CODE REVIEW

File: modules/cms-kit/src/Volo.CmsKit.Domain/Volo/CmsKit/Comments/CommentManager.cs [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        }
+
+        public virtual async Task<Comment> CreateAsync([NotNull] CmsUser creator,
+                                                       [NotNull] string entityType,
+                                                       [NotNull] string entityId,
+                                                       [NotNull] string text,
+                                                       [CanBeNull] Guid? repliedCommentId = null)
+        {
+            Check.NotNull(creator, nameof(creator));
+
```
Comment (ilkayilknur) [inline modules/cms-kit/src/Volo.CmsKit.Domain/Volo/CmsKit/Comments/CommentManager.cs:30]: We should add null checks for entityType, entityId, and text as well.

**Review by ilkayilknur — CHANGES_REQUESTED** [review]
I have small feedback about null checks. Rest of the code LGTM 👍

File: modules/cms-kit/src/Volo.CmsKit.Domain/Volo/CmsKit/Comments/EntityNotCommentableException.cs [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.Serialization;
+using System.Text;
+using System.Threading.Tasks;
+using Volo.Abp;
+
+namespace Volo.CmsKit.Comments
+{
+    public class EntityNotCommentableException : BusinessException
```
Comment (cotur) [inline modules/cms-kit/src/Volo.CmsKit.Domain/Volo/CmsKit/Comments/EntityNotCommentableException.cs:12]: Requires `[Serializable]` attribute

File: modules/cms-kit/src/Volo.CmsKit.Domain/Volo/CmsKit/Comments/CommentManager.cs (hunk 1 above)
Comment (enisn) [reply modules/cms-kit/src/Volo.CmsKit.Domain/Volo/CmsKit/Comments/CommentManager.cs:30]: Those are checked in constructor of Comment entity. That's why I ignored from here. Should we ad here too? @ilkayilknur

Comment (ilkayilknur) [reply modules/cms-kit/src/Volo.CmsKit.Domain/Volo/CmsKit/Comments/CommentManager.cs:30]: Yes, we should add it. The implementation of the might change in the future. We also check the entityType definition below. What happens if the entityType is null? We should add checks.

Comment (enisn) [reply modules/cms-kit/src/Volo.CmsKit.Domain/Volo/CmsKit/Comments/CommentManager.cs:30]: Ok, it's done with 96e84af

**Review by ilkayilknur — APPROVED** [review]

**Comment by enisn** [comment]
Awaiting to find possible breaks

**Pull request merged**